README.md

```diff
@@ -101,6 +101,8 @@ These builds are maintained by the community. While they should be safe, downloa
 
 * Windows Package Manager (winget): [Usage](https://docs.microsoft.com/en-us/windows/package-manager/winget/)
 
+* Homebrew Formulae (Mac only):[Download](https://formulae.brew.sh/cask/freetube)
```

Comment:
Could you move this after FreeTubeCordova? We keep the unofficial downloads in alphabetical order

Reply:
Hi @ChunkyProgrammer , moved Homebrew after Cordova. Should be good now